### PR TITLE
Hotfix/prestashop validator compat

### PR DESCRIPTION
**Pull request by hannunosto**

### CODE REVIEW

**Comment by mridang** [comment]
Add `build.xml` to gitattributes.

**Comment by mridang** [comment]
Ensure `644` permissions.

File: config.xml [hunk 1]
```diff
@@ -27,7 +27,8 @@
 	<name>nostotagging</name>
 	<displayName><![CDATA[Personalization for PrestaShop]]></displayName>
 	<version><![CDATA[2.5.0]]></version>
-	<description><![CDATA[Increase your conversion rate and average order value by delivering your customers personalized product recommendations throughout their shopping journey.]]></description>
+	<description><![CDATA[Increase your conversion rate and average order value by delivering your customers personalized product
+            recommendations throughout their shopping journey.]]></description>
```
Comment (mridang) [inline config.xml]: New line.

**Comment by mridang** [comment]
I think autloader should excluded.

**Comment by mridang** [comment]
Remove the `index.php` files.

**Comment by mridang** [comment]
Looks good.

File: config.xml (hunk 1 above)
Comment (hannunosto) [inline config.xml]: This is required by the Prestashop validator. Needs to match exactly what's in module's bootstrap file

**Pull request merged**